packages/base/src/3dview/mainview.tsx

```diff
@@ -423,6 +423,20 @@ export class MainView extends React.Component<IProps, IStates> {
 
         const updatedPosition = new THREE.Vector3();
         updatedObject.getWorldPosition(updatedPosition);
+        const updatedQuaternion = new THREE.Quaternion();
+        updatedObject.getWorldQuaternion(updatedQuaternion);
+
+        const s = Math.sqrt(1 - updatedQuaternion.w * updatedQuaternion.w);
+        const updatedRotation = [
+          [
+            parseFloat((updatedQuaternion.x / s).toFixed(10)),
```

Comment:
I suspect the error comes from `parseFloat` which returns may `NaN`. Do we really need `parseFloat` here? Why not just `updatedQuaternion.x / s`?

Reply:
Probably a division by `0` when s is `0`, there should be a check for this

Reply:
> Do we really need `parseFloat` here? Why not just `updatedQuaternion.x / s`?

Yes, it is also sufficient.

> Probably a division by 0 when s is 0, there should be a check for this

Added a fallback condition, thanks :)